.gitlab-ci.yml

```diff
@@ -0,0 +1,164 @@
+default:
+  image: docker-io.artifactory.pan.local/devdemisto/gitlab-content-ci:1.0.0.18863
+  cache:
+    key:
+      files:
+       - "dev-requirements-py3.txt"
+       - "package-lock.json"
+    paths:
+      - venv/
+      - node_modules/
+    when: always
+  before_script:
+    - echo "=== Running before script ==="
+    - git checkout $CI_COMMIT_BRANCH
+    - mkdir -p -m 777 $ARTIFACTS_FOLDER/
+    - |
+      if [[ -f "$BASH_ENV" ]]; then
+        source "$BASH_ENV"
+      fi
+    - echo "=== Creating new clean logs folder ==="
+    - rm -rf $ARTIFACTS_FOLDER/logs
+    - mkdir -p $ARTIFACTS_FOLDER/logs
+    - echo "=== Granting execute permissions on files ==="
+    - chmod +x ./Tests/scripts/*
+    - chmod +x ./Tests/lastest_server_build_scripts/*
```

Comment:
I think we can stop supporting this folder (the flow doesn't work anymore).

need to document the alternative option (create a temporary AMI)